components/button/stories/button.stories.js

```diff
@@ -115,3 +118,171 @@ WithForcedColors.parameters = {
+ * action requires less prominence, or if there are multiple primary actions of the same importance
+ * in the same view. Both the fill (default) and outline styles are demonstrated in this example.
+ */
+export const Primary = TreatmentTemplate.bind({});
+Primary.tags = ["!dev"];
+Primary.args = {
+	variant: "primary",
+	treatmentLayout: "stacked",
+};
+Primary.parameters = {
+	chromatic: { disableSnapshot: true },
+};
+
+/**
+ * The secondary button is for low emphasis. It’s paired with other button types to surface less
+ * prominent actions, and should never be the only button in a group. Both the fill (default) and
+ * outline styles are demonstrated in this example.
+ */
+export const Secondary = TreatmentTemplate.bind({});
+Secondary.tags = ["!dev"];
+Secondary.args = {
+	variant: "secondary",
+	treatmentLayout: "stacked",
+};
+Secondary.parameters = {
+	chromatic: { disableSnapshot: true },
+};
+
+/**
+ * The negative button is for emphasizing actions that can be destructive or have negative
+ * consequences if taken. Use it sparingly.
+ */
+export const Negative = ButtonsWithIconOptions.bind({});
+Negative.tags = ["!dev"];
+Negative.args = {
+	variant: "negative",
+};
+Negative.parameters = {
+	chromatic: { disableSnapshot: true },
+};
+
+export const StaticWhitePrimary = TreatmentTemplate.bind({});
+StaticWhitePrimary.tags = ["!dev"];
+StaticWhitePrimary.args = {
+	variant: "primary",
+	treatmentLayout: "stacked",
+	staticColor: "white",
+};
+StaticWhitePrimary.parameters = {
+	chromatic: { disableSnapshot: true },
+};
+
+export const StaticWhiteSecondary = TreatmentTemplate.bind({});
+StaticWhiteSecondary.tags = ["!dev"];
+StaticWhiteSecondary.args = {
+	variant: "secondary",
+	treatmentLayout: "stacked",
+	staticColor: "white",
+};
+StaticWhiteSecondary.parameters = {
+	chromatic: { disableSnapshot: true },
+};
+
+export const StaticBlackPrimary = TreatmentTemplate.bind({});
+StaticBlackPrimary.tags = ["!dev"];
+StaticBlackPrimary.args = {
+	variant: "primary",
+	treatmentLayout: "stacked",
+	staticColor: "black",
+};
+StaticBlackPrimary.parameters = {
+	chromatic: { disableSnapshot: true },
+};
+
+export const StaticBlackSecondary = TreatmentTemplate.bind({});
+StaticBlackSecondary.tags = ["!dev"];
+StaticBlackSecondary.args = {
+	variant: "secondary",
+	treatmentLayout: "stacked",
+	staticColor: "black",
+};
+StaticBlackSecondary.parameters = {
+	chromatic: { disableSnapshot: true },
+};
+
+/**
+ * The pending button is for indicating that a quick progress action is taking place. In this case, the
+ * label and optional icon disappear and a progress circle appears. The progress circle always shows an
+ * indeterminate progress. We recommend the use of the `.is-pending` class on the component’s parent
+ * container, but there is also an option to use an attribute of `pending` instead. Buttons should have
+ * the disabled attribute when the pending state is applied.
+ */
+export const Pending = ButtonsWithIconOptions.bind({});
```

Comment:
Can Pending have an outline treatment? I didn't see anything about it being removed for S2. That is something that could be shown or mentioned here.

I have the same question about Disabled but that story is right below this one so I won't duplicate the question.

My assumption would be that if the button has an outline treatment (like the primary/secondary), if it was disabled or pending it would have the same treatment?